### DISCUSSION TITLE
Thanks for all the work!

### DISCUSSION THREAD

**Discussion by Hirogen** [SebLague/Geographical-Adventures#27] [Show and tell]
Thanks for all the work you do, you are a blessing to the game dev community.

Is the City-Mini-Game also in the source? Or will that be released later. I'm super curious how you did the OSM Landscape and City Creation 🥇 .

**Comment by SebLague** [comment]
Hello, I'm happy you like the project! I'm afraid the skywriting experiment is currently only available to patrons of the channel (over [here](https://www.patreon.com/posts/source-code-for-65893075)), but I'll make it public in a few weeks. The city creation was done using this blender [osm plugin](https://prochitecture.gumroad.com/l/blender-osm).

**Reply by Hirogen** [reply]
Thanks :)

**Comment by punyidea** [comment]
Thank you!! In its current state the game is quite enjoyable to play :) I also appreciate the community for the super quick linux shader fix which affected me. In short, you rock!
